components/AppBar/src/MDCAppBarViewController.m

```diff
@@ -246,6 +246,10 @@ - (void)didMoveToParentViewController:(UIViewController *)parent {
   [super didMoveToParentViewController:parent];
 
   [self.navigationBar observeNavigationItem:parent.navigationItem];
+
+  CGRect frame = self.view.frame;
+  frame.size.width = parent.view.bounds.size.width;
```

Comment:
We need to use CGRectGetWidth() if we are using non-standardized CGRects.

Reply:
Done